app/build.gradle

```diff
@@ -3,9 +3,8 @@ apply plugin: 'com.android.application'
 android {
     compileSdkVersion 24
     buildToolsVersion "23.0.3"
-
     defaultConfig {
-        applicationId "lnmiit.com.lnmiit"
+        applicationId 'lnmiit.com.madclub'
```

Comment:
rename this package to something logical like org.lnmiitapp or something similar as mad club is in merging process with cybros.